internal/validations/clusterlogforwarder/validate_clusterlogforwarderspec_test.go

```diff
@@ -284,6 +285,20 @@ var _ = Describe("Validate clusterlogforwarderspec", func() {
 			clfStatus = &loggingv1.ClusterLogForwarderStatus{}
 			extras = map[string]bool{}
 
+			cloudWatchSecret = &corev1.Secret{
+				TypeMeta: metav1.TypeMeta{
```

Comment:
We have a factory in the "runtime" package that may make this easier and possibly a matching builder